docs/index.md

```diff
@@ -224,6 +226,57 @@ resource "grafana_oncall_escalation" "example_notify_step" {
 
+### Managing Connections
+
+#### Obtaining Connections access token
+
+Before using the Terraform Provider to manage Grafana Connections resources, such as metrics endpoint scrape jobs, you need to create an access policy token on the Grafana Cloud Portal. This token is used to authenticate the provider to the Grafana Connections API.
+[These docs](https://grafana.com/docs/grafana-cloud/account-management/authentication-and-permissions/access-policies/authorize-services/#create-an-access-policy-for-a-stack) will guide you on how to create
+an access policy. The required permissions, or scopes, are `integration-management:read`, `integration-management:write` and `stacks:read`.
+
+Also, by default the Access Policies UI will not show those scopes, to find name you need to use the `Add Scope` textbox, as shown in the following image:
+
+<img src="https://grafana.com/media/docs/grafana-cloud/aws/cloud-provider-terraform-access-policy-creation.png" width="700"/>
+
+1. Use the `Add Scope` textbox (1) to search for the permissions you need to add to the access policy.
+1. Make sure that you configure the three required scopes. Once done, you'll see the selected scopes as in (2).
+
+Having created an Access Policy, you can now create a token that will be used to authenticate the provider to the Connections API. You can do so just after creating the access policy, following
+the in-screen instructions, of following [this guide](https://grafana.com/docs/grafana-cloud/account-management/authentication-and-permissions/access-policies/authorize-services/#create-one-or-more-access-policy-tokens).
+
+#### Obtaining Connections API hostname
+
+Having created the token, we can find the correct Connections API hostname by running the following script, that requires `curl` and [`jq`](https://jqlang.github.io/jq/) installed:
+
+```bash
+curl -sH "Authorization: Bearer <Access Token from previous step>" "https://grafana.com/api/instances" | \
+  jq '[.items[]|{stackName: .slug, clusterName:.clusterSlug, connectionsAPIURL: "https://connections-api-\(.clusterSlug).grafana.net"}]'
+```
+
+This script will return a list of all the Grafana Cloud stacks you own, with the Connections API hostname for each one. Choose the correct hostname for the stack you want to manage.
+For example, in the following response, the correct hostname for the `examplestackname` stack is `https://connections-api-prod-eu-west-0.grafana.net`.
+
+```json
+[
+  {
+    "stackName": "examplestackname",
+    "clusterName": "prod-eu-west-0",
+    "connectionsAPIURL": "https://connections-api-prod-eu-west-0.grafana.net"
+  }
+]
+```
+
+#### Configuring Provider
+
+Once you have the token and Connections API hostname, you can configure the provider as follows:
+
+```hcl
+provider "grafana" {
+  connections_api_url          = "<Connections API URL from previous step>"
+  connections_api_access_token = "<Access Token from previous step>"
+}
+```
+
 ## Authentication
```

Comment:
Can we add `connections_api_access_token` to this section, as was done for https://github.com/grafana/terraform-provider-grafana/blob/6e384ec04b41e7dd77b8aaa5978125721fd3b763/docs/index.md?plain=1#L380

Reply:
Added in 88c37be89d0be8b564bf3000b54b11582bb5672f